update REAMDME with jobs API (#726)
gordon-cs/gordon-360-api: README.md

`apartment/applications/{applicationID}` Deletes an application (and consequently all rows that reference it)

### Jobs

> What is it? Resource that represents the shifts regarding a user.

Reviewer: ```suggestion
What is it? Resource to interact with a student's timesheets for on-campus jobs.
```

##### GET

`api/jobs` Get the active jobs of the current logged in user.
`api/jobs/getSavedShift` Get the saved shifts of the current logged in user.
`api/jobs/supervisorName/{supervisorID}` Get the name of a supervisor based on their ID number `supervisorID` as a parameter.

> `api/jobs/clockOut` Get the clock in the status from the backend of the current logged in user, true if user is clocked in, and false if clocked out.

Reviewer: ```suggestion
`api/jobs/clockOut` Get the time-clock status of the current logged in user, true if user is clocked in, and false if clocked out.
```

##### POST

`api/jobs/saveShift` Create a new shift for the current logged in user.
`api/jobs/submitShifts` Submit all shifts for the current logged in user.

> `api/jobs/clockIn` Send the current clock in status to the backend for the current logged in user, true if user is clocked in, and false if clocked out.

Reviewer: ```suggestion
`api/jobs/clockIn` Update the time-clock status of the current logged in user, true if user is clocked in, and false if clocked out.
```
